### PR TITLE
feat: Add an enable-rtcstats field to ColibriConferenceIQ.

### PR DESCRIPTION
**Pull request by gpolitis**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -101,7 +101,7 @@
     <dependency>
       <groupId>${project.groupId}</groupId>
       <artifactId>jitsi-xmpp-extensions</artifactId>
-      <version>1.0-30-g48113ca</version>
+      <version>1.0-SNAPSHOT</version>
```
Comment (damencho) [inline pom.xml:104]: I guess this is temporary?

Comment (gpolitis) [reply pom.xml:104]: yep

File: src/main/java/org/jitsi/jicofo/conference/colibri/ColibriConferenceImpl.java [hunk 2]
```diff
@@ -125,6 +125,16 @@ public void setGID(String gid)
         conferenceState.setGID(gid);
     }
 
+    @Override
+    public void setRtcStatsEnabled(boolean rtcStatsEnabled) {
+        conferenceState.setRtcStatsEnabled(rtcStatsEnabled);
```
Comment (bgrozev) [inline src/main/java/org/jitsi/jicofo/conference/colibri/ColibriConferenceImpl.java:130]: Have you tested that this? The code is super hairy and hard to follow, but I think the requests go through `colibriBuilder`, which gets reset at certain times, and `reset()` doesn't re-read the new flags

Comment (gpolitis) [reply src/main/java/org/jitsi/jicofo/conference/colibri/ColibriConferenceImpl.java:130]: I just did and it doesn't work :-( I guess you nailed the problem.. I'll dig into it.

**Comment by gpolitis** [comment]
Blocked by https://github.com/jitsi/jicofo/pull/832

**Review by bgrozev — APPROVED** [review]
